bin/app.dart

```diff
@@ -0,0 +1,92 @@
+/**
+ * Created by lejard_h on 06/05/16.
+ */
+
+import "dart:io";
+import 'dart:async';
+
+import "package:dev_string_converter/dev_string_converter.dart";
+import "utils.dart";
+
+
+StreamSubscription _progressSubscription;
+
+void _progress() {
+    _progressSubscription =
+        new Stream.periodic(const Duration(seconds: 1)).listen((_) {
+            output('.', Color.white);
+        });
+}
+
+void _endProgress() {
+    _progressSubscription.cancel();
+    stdout.write('\n');
+}
+
+
+main(List<String> args) async {
+    var name = toTableName(args[0]);
+    output('Cloning template repository', Color.green);
+    _progress();
+    ProcessResult result;
+    try {
+        result = await Process.run('git', [
+            'clone',
+            'https://github.com/lejard-h/angular2_project_template.git',
+            '-bmaster',
+            '--single-branch',
+            name,
+        ]);
```

Comment:
I don't like the fact of cloning an external repository. This looks a little bit difficult to maintain and it could cause security problems. I would prefer to have a template that is inside the application.

Another think is that it doesn't promote the reuse of code. I mean we already created templates for components, pipes, services, etc.